src/hooks/sarcoToken/useAllowance.ts

```diff
@@ -0,0 +1,15 @@
+import { useAccount, useContractRead } from 'wagmi';
+import { SarcoTokenABI } from '../../abis/SarcoToken';
+
+export function useAllowance() {
+  const { address } = useAccount();
+
+  const { data } = useContractRead({
+    addressOrName: process.env.REACT_APP_SARCO_TOKEN_ADDRESS || '',
+    contractInterface: SarcoTokenABI,
+    functionName: 'allowance',
+    args: [address, process.env.REACT_APP_LOCAL_CONTRACT_ADDRESS],
```

Comment:
Can this env variable be updated so that we won't have to remember to change it when migrating to non-local environments?

Maybe just literally nuke the whole "LOCAL_VARIABLE" concept (since realistically we're not testing on both local _and_ non-local environments simultaneously).

Or add `.env` to git and make use of the `.env.local` pattern, which apparently is the recommended approach for react apps anyways -- sth that Adam suggested I did with the FrostByte app 
(not sure if this is the best reference but: https://create-react-app.dev/docs/adding-custom-environment-variables/)

Reply:
Issue #17 will handle this situation. I think my solution will be more flexible than .env.local since Rainbowkit supports hardhat for the localnetwork. So using the app for production, testing, and local should work all the same.

Reply:
Good suggestion. We should make this a new task.

Reply:
Nevermind, didn't see Eric's comment when I wrote this